src/main/snapshot.ts

```diff
@@ -186,7 +188,11 @@ export async function downloadSnapshot(
     });
     savingPaths = await Promise.all(downloadPromise);
   } catch (error) {
-    throw new DownloadSnapshotFailedError(basePath, savingPaths.join(", "));
+    if ((error = "Error: Clear cache requested.")) {
```

Comment:
Is it intended?

```suggestion
    if (error === "Error: Clear cache requested.") {
```

Reply:
@longfin Thanks for pointing it out! I realized that I also made a mistake on parsing the error message so I made more corrections. PTAL 🙏